components/KiboShipping.vue

```diff
@@ -6,13 +6,44 @@
       class="sf-heading--left sf-heading--no-underline title"
     />
     <div class="user-addresses">
-      <UserSavedAddresses
-        :countries="countries"
-        :default-address="shippingAddress"
-        :addresses="userShippingAddresses"
-        :is-readonly="isReadonly"
-        @onSave="saveShippingAddress"
-        @validateForm="validateShippingDetails"
+      <SfLoader :loading="loadingSavedUserAddress">
+        <UserSavedAddresses
+          :countries="countries"
+          :default-address="shippingAddress"
+          :addresses="userShippingAddresses"
+          :is-readonly="isReadonly"
+          @onSave="saveShippingAddress"
+          @validateForm="validateShippingDetails"
+          @toggleAddressFormVisibility="toggleAddressFormVisibility"
+        />
+      </SfLoader>
+    </div>
+
+    <div v-if="isAddressFormOpen && isUserLoggedIn">
+      <SfCheckbox
+        name="shipping"
+        label="Save shipping address"
+        hint-message=""
+        :required="false"
+        info-message=""
+        error-message=""
+        valid
+        :disabled="false"
+        :selected="isSaveShippingAddresChecked"
+        @change="changeAddressSave"
+      />
+
+      <SfCheckbox
+        name="shipping"
+        label="Make this my default address"
```

Comment:
Same, internationalization should be used here.

Reply:
Resolved